test/custom-levels.test.js

```diff
@@ -244,3 +244,20 @@ test('custom level does not affect changeLevelName', async ({ is }) => {
   const { priority } = await once(stream, 'data')
   is(priority, 35)
 })
+
+test('default levels can be redefined using custom levels', async ({ is }) => {
+  const stream = sink()
+  const logger = pino({
+    customLevels: {
+      info: 35,
+      debug: 45
+    },
+    useOnlyCustomLevels: true
+  }, stream)
+
+  is(logger.hasOwnProperty('info'), true)
+
+  logger.info('test')
+  const { priority } = await once(stream, 'data')
```

Comment:
This is missing the `changeLevelName` option.

Reply:
Ah… Then I misunderstood the `changeLevelName` option. And as I only want to test that my level setting is taken over, I will just use the standard name.